src/docfx/Errors.cs

```diff
@@ -475,6 +475,20 @@ public static Error FieldDeprecated(SourceInfo source, string name, string repla
         public static Error ValuesNotMatch(SourceInfo source, string name, object value, string dependentFieldName, object dependentFieldValue, IEnumerable<object> validValues)
             => new Error(ErrorLevel.Warning, "values-not-match", $"Invalid value for {name}: '{value}' is not valid with '{dependentFieldName}' value '{dependentFieldValue}'. Valid values: {Join(validValues)}", source);
 
+        /// <summary>
+        /// The value is not a valid Microsoft alias
+        /// </summary>
+        /// Behavior: ✔️ Message: ❌
+        public static Error MsAliasInvalid(SourceInfo source, string name, string alias)
+            => new Error(ErrorLevel.Warning, "ms-alias-invalid", $"Invalid value for '{name}', '{alias}' is not a valid Microsoft alias", source);
+
+        /// <summary>
+        /// The value is not a valid Microsoft alias
+        /// </summary>
+        /// Behavior: ✔️ Message: ❌
+        public static Error GraphApiGetUsersFailed(string exMessage)
```

Comment:
consider just call it `microsoft-graph-api-failed`
```suggestion
        public static Error MicrosoftGraphApiFailed(string exMessage)
```